exporters/zipkin/zipkin.go

```diff
@@ -79,7 +81,8 @@ func WithClient(client *http.Client) Option {
 // New creates a new Zipkin exporter.
 func New(collectorURL string, opts ...Option) (*Exporter, error) {
 	if collectorURL == "" {
-		return nil, errors.New("collector URL cannot be empty")
+		// Use endpoint from env var or default collector URL.
+		collectorURL = envOr(envEndpoint, defaultCollectorURL)
```

Comment:
If we are overriding this, shall we add a info log explaining where this come from?